controller.rst

```diff
@@ -286,6 +286,45 @@ in your controllers.
 
 For more information about services, see the :doc:`/service_container` article.
 
+Autowire Parameter Attribute
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 6.1
+
+    The ``#[Autowire]`` attribute was introduced in Symfony 6.1.
+
+Services that cannot be autowired, :ref:`parameters <service-parameters>` and even
+:doc:`complex expressions </service_container/expression_language>` can be bound
+to a controller argument with the ``#[Autowire]`` attribute::
+
+    use Psr\Log\LoggerInterface;
+    use Symfony\Component\DependencyInjection\Attribute\Autowire;
+    use Symfony\Component\HttpFoundation\Response;
+    // ...
+
+    /**
+     * @Route("/lucky/number/{max}")
```

Comment:
Maybe something to do in another PR, but why don't we use the attribute style? Especially with this feature from Symfony 6.1

Reply:
Agree, was just following the method from the other examples in this file. Probably a separate PR is best.